apps/hash-ai-worker-py/app/typesystem/property_type.py

```diff
@@ -26,3 +74,34 @@ class PropertyTypeSchema(OntologyTypeSchema, OneOf[PropertyValue]):
     """
 
     kind: Literal["propertyType"]
+
+    async def create_property_type(
+        self,
+        *,
+        actor_id: UUID,
+    ) -> Annotated[Any, "PropertyTypeAnnotation"]:
```

Comment:
I don't like the escape into `Any`, but I don't see any obvious way to do it otherwise.